### ISSUE TITLE
can't use Faerie Fire,ghost when press ALT

### ISSUE THREAD

**Issue by Avarise831** [ValveSoftware/Dota2-Gameplay#6591]
### Item name

Faerie Fire,Ghost Scepter etc

### Description

![1233](https://user-images.githubusercontent.com/37029303/212273662-1462b672-320d-4179-ba44-e6465bd736dd.png)
can't use Faerie Fire,ghost when pressed ALT "No target"

### Example Match ID and Timestamp

-

**Comment by shaileshgarg** [comment]
Makes sense right, Alt + 3, or Alt + 4 can be bound for some other action.

"No target" probably in your case they are bound to something with unit target.

**Comment by Avarise831** [comment]
> Makes sense right, Alt + 3, or Alt + 4 can be bound for some other action.
> 
> "No target" probably in your case they are bound to something with unit target.

I understand what you're talking about, but my settings fine. This bug has been around since the release of Faerie Fire the game thinks that it can be transferred, but such a function does not exist same problem with ghots but etheral work fine.
in the game I often have to press alt to see the icon on the minimap,range towers,range camps its just ruin my game expiriance when i cant use my items correct

**Comment by shaileshgarg** [comment]
Are you able to use other items, like Hood of Definace, Drums with Alt Pressed?

**Comment by Avarise831** [comment]
> Are you able to use other items that are self target, like Hood of Definace, Drums with Alt Pressed?
> 
> I can't use any with Alt Pressed.

actually I think this  bug  only with Faerie Fire "No Target"
